incompat/early-initializer/src/main/scala-2.13/early-initializer.scala

```diff
@@ -0,0 +1,6 @@
+trait Bar {
+  val name: String
+  val size: Int = name.size
+}
+
+object Foo extends { val name = "Foo" } with Bar
```

Comment:
Consider formatting this is the more widely used way:
```suggestion
object Foo extends {
  val name = "Foo"
} with Bar
```